app/src/controllers/step/actionValidators.js

```diff
@@ -17,3 +17,10 @@ export const validateIncludeInAA = (item) => {
   }
   return null;
 };
+
+export const validateLinkIssue = (item) => {
```

Comment:
Actually, name of function a bit confusing. It called `validate*`, but just return `null` or some string and don't really validate something.

Reply:
I named it according to all our action validators defined for `groupOperaions`.
Common logic of validation locates at `app/src/controllers/groupOperations`